src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -69,7 +69,7 @@ fi
 
 if [ "$EXECUTION_DIR" != "$BASEDIR" ]; then
   echo "Copying files into execution dir."
-  ln -f $BASEDIR/* $EXECUTION_DIR || exit -1
+  ln -f $BASEDIR/* $EXECUTION_DIR || echo "Link failed, fallback to copy"; cp -run $BASEDIR/* $EXECUTION_DIR || echo "Still errors, attempting to continue"
```

Comment:
I'm fine with the fallback but I would still exit -1 if the copy fails.

Depending on how often this occurs it might also be worth eliminating the echo of "Link failed" if it ends up causing a bunch of noise, but we can leave it for now.